src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -119,12 +119,8 @@ In this step, you'll start the LDR job from the destination cluster. You can rep
 
 _Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
 
-- `immediate` (default): Attempts to replicate the changed row directly into the destination table, without re-running constraint validations. It does not support writing into tables with [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) constraints.
-- `validated`: Attempts to apply the write in a similar way to a user-run query, which would re-run all constraint validations relevant to the destination table(s). If the change violates foreign key dependencies, unique constraints, or other constraints, the row will be put in the [dead letter queue (DLQ)]({% link {{ page.version.version }}/manage-logical-data-replication.md %}#dead-letter-queue-dlq) instead.
-
```

Comment:
For now, I deleted the following note on the tutorial page re the TTL deletes option. Mainly because there was some discussion when this page was published as to whether we should include the note. Now that we're marking the option as unidirectional only, it seems better to maybe remove. However, happy to add this back in, lmk during review.